crypto/transactions/deserializer.py

```diff
@@ -104,8 +105,8 @@ def _handle_version_one(self, transaction):
             )
 
         if transaction.type is TRANSACTION_MULTI_SIGNATURE_REGISTRATION:
-            transaction.asset['multisignature']['keysgroup'] = [
-                '+{}'.format(key) for key in transaction.asset['multisignature']['keysgroup']
+            transaction.asset['multiSignature']['publicKeys'] = [
+                '+{}'.format(key) for key in transaction.asset['multiSignature']['publicKeys']
```

Comment:
this looks like old multisig

Reply:
seems it's inside the `handle_version_one` function, which can be removed anyway